app/app.rb

```diff
@@ -0,0 +1,76 @@
+ENV["RACK_ENV"] ||= "development"
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative './models/peep.rb'
+require_relative './models/user.rb'
+require_relative 'data_mapper_setup'
+
+class Chitter < Sinatra::Base
+  use Rack::MethodOverride
+
+  enable :sessions
+  set :session_secret, 'super secret'
+  register Sinatra::Flash
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :'links/index'
+  end
+
+  post '/peeps/new' do
+    Peep.create(content: params[:new_peep], creator: params[:username])
+    redirect to '/peeps'
+  end
+
+  get '/peeps/new' do
+    erb :'links/add_peep'
+  end
+
+  get '/users/new' do
+    @user = User.new
+    erb :'users/new'
+  end
+
+  post '/users' do
+    @user = User.create(email: params[:email],
+                       name: params[:name],
+                       username: params[:username],
+                       password: params[:password],
+                       password_confirmation: params[:password_confirmation])
+    if @user.save
+      session[:user_id] = @user.id
```

Comment:
Can this line be made more declarative of what it does? At the moment, it seems to reveal its implementation, rather than its intention.